mav_planning_msgs/srv/PlannerService.srv

```diff
@@ -8,7 +8,8 @@ geometry_msgs/Vector3 bounding_box
 # True on success, false on planning failure
 bool success
 # Either contains a polynomial trajectory:
-mav_planning_msgs/PolynomialTrajectory4D polynomial_plan
+mav_planning_msgs/PolynomialTrajectory polynomial_plan
+mav_planning_msgs/PolynomialTrajectory4D polynomial_plan_4D
```

Comment:
Woudn't this break downward compatibility? I'll test this.

Reply:
It would break where there is use of "polynomial_plan" in the service call. We could instead create a new name for the non-4d trajectory.